src/calculateRentalCost.js

```diff
@@ -34,17 +34,15 @@ function calculateRentalCost(days) {
 
   const count = days * price;
 
-  if (days < 3) {
-    return count;
-  }
-
-  if (days < 7) {
+  if (days >= 3 && days < 7) {
```

Comment:
There's no need to use such a complex condition. You can check first `if (days >= 7)` and after that you can use `if (days >= 3)` condition because all numbers greater than `7` don't pass through first condition

Reply:
i have understand this after push first fixing commit, so check the second fixed commit, please